fragment.go

```diff
@@ -1732,8 +1732,10 @@ func (f *fragment) importRoaring(data []byte, clear bool) error {
 	rowSet := make([]uint64, 0)
 	var lastRow uint64 = math.MaxUint64
 
+	incomingCnt := 0
 	for iter.Next() {
-		key, _ := iter.Value()
+		key, c := iter.Value()
+		incomingCnt += int(c.N())
```

Comment:
hmm. should this be int64? what if someone wants to import 2 billion bits all at once?